SPEC.md

```diff
@@ -104,44 +106,42 @@ The test index makes use of symbolic language in describing connection and messa
     <- pong response with the same `sequence` number
 
 ### ZG-CONFORMANCE-004
-    
+
     The node responds with mtLEDGER_DATA for mtGET_LEDGER with iType == LiBase.
-    
+
     <>
     -> mtGET_LEDGER with iType == LiBase
     <- mtLEDGER_DATA
 
-### ZG-CONFORMANCE-005
 
     The node responds with mtLEDGER_DATA for mtGET_LEDGER with iType == LiAsNode.
 
     <>
     -> mtGET_LEDGER with iType == LiAsNode
     <- mtLEDGER_DATA
 
-### ZG-CONFORMANCE-006
+### ZG-CONFORMANCE-005
 
     The node requests mtGET_PEER_SHARD_INFO_V2 after connection and handshake.
 
     <>
     <- mtGET_PEER_SHARD_INFO_V2
 
-### ZG-CONFORMANCE-007
 
     The node should *NOT* request mtGET_PEER_SHARD_INFO_V2 after connection if there was no handshake.
```

Comment:
This is different behaviour, I'd suggest leaving this as a separate case to the new 005.

Reply:
Sure. I initially did this split because the same message was mentioned.